fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HMSExternalTable.java

```diff
@@ -654,6 +654,9 @@ public long getLastDdlTime() {
     public Optional<SchemaCacheValue> initSchema(SchemaCacheKey key) {
         makeSureInitialized();
         if (dlaType.equals(DLAType.ICEBERG)) {
+            if (isView()) {
+                throw new RuntimeException("Cannot use ICEBERG schema when view is used");
```

Comment:
So if I execute `describe iceberg_view`, I will get this error?
And I see in `IcebergExternalTable`, the `initSchema()` can work with view, which is not consistent with logic here.